specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/types.json

```diff
@@ -70,250 +185,212 @@
       "type": "string",
+      "pattern": "^[0-9A-Fa-f]{8}-([0-9A-Fa-f]{4}-){3}[0-9A-Fa-f]{12}$",
       "x-ms-parameter-location": "method"
     },
-    "billingAccountName": {
-      "description": "The ID that uniquely identifies a billing account.",
+    "billingProfileName": {
       "in": "path",
-      "name": "billingAccountName",
+      "name": "billingProfileName",
+      "description": "The ID that uniquely identifies a billing profile.",
       "required": true,
       "type": "string",
+      "pattern": "^[a-zA-Z\\d-_]{1,128}$",
       "x-ms-parameter-location": "method"
     },
-    "billingProfileName": {
-      "description": "The ID that uniquely identifies a billing profile.",
+    "invoiceSectionName": {
       "in": "path",
-      "name": "billingProfileName",
+      "name": "invoiceSectionName",
+      "description": "The ID that uniquely identifies an invoice section.",
       "required": true,
       "type": "string",
+      "pattern": "^[a-zA-Z\\d-_]{1,128}$",
       "x-ms-parameter-location": "method"
     },
     "customerName": {
-      "description": "The ID that uniquely identifies a customer.",
       "in": "path",
       "name": "customerName",
+      "description": "The ID that uniquely identifies a customer.",
       "required": true,
       "type": "string",
+      "pattern": "^[a-zA-Z\\d-_]{1,128}$",
       "x-ms-parameter-location": "method"
     },
-    "invoiceName": {
-      "description": "The ID that uniquely identifies an invoice.",
+    "subscriptionId": {
       "in": "path",
-      "name": "invoiceName",
+      "name": "subscriptionId",
+      "description": "The ID that uniquely identifies a billing subscription.",
       "required": true,
       "type": "string",
+      "pattern": "^[0-9A-Fa-f]{8}-([0-9A-Fa-f]{4}-){3}[0-9A-Fa-f]{12}$",
       "x-ms-parameter-location": "method"
     },
-    "invoiceSectionName": {
-      "description": "The ID that uniquely identifies an invoice section.",
+    "billingRequestName": {
       "in": "path",
-      "name": "invoiceSectionName",
+      "name": "billingRequestName",
+      "description": "The ID that uniquely identifies a billing request.",
       "required": true,
       "type": "string",
+      "pattern": "^[0-9A-Fa-f]{8}-([0-9A-Fa-f]{4}-){3}[0-9A-Fa-f]{12}$",
       "x-ms-parameter-location": "method"
     },
-    "subscriptionId": {
-      "description": "The ID that uniquely identifies an Azure subscription.",
+    "billingSubscriptionName": {
       "in": "path",
-      "name": "subscriptionId",
+      "name": "billingSubscriptionName",
+      "description": "The ID that uniquely identifies a billing subscription.",
+      "required": true,
+      "type": "string",
+      "pattern": "^[0-9A-Fa-f]{8}-([0-9A-Fa-f]{4}-){3}[0-9A-Fa-f]{12}$",
+      "x-ms-parameter-location": "method"
+    },
+    "enrollmentAccountName": {
+      "in": "path",
+      "name": "enrollmentAccountName",
+      "description": "The name of the enrollment account.",
```

Comment:
This looks incosistent with others - billingSubscriptionName and billingRequestName. Shouldn't this be enrollmentAccountDisplayName if the property provides the display name.

Reply:
Updated the description to correct one "The ID that uniquely identifies an enrollment account.". Can you please review this change @amberbhargava .